NZSLDict/Classes/SearchViewController.swift

```diff
@@ -350,7 +373,7 @@ class SearchViewController: UIViewController, UISearchBarDelegate, UITableViewDa
         if sender.state == .Ended {
             self.selectEntry(wordOfTheDay)
             searchBar.resignFirstResponder()
-            self.delegate.didSelectEntry(wordOfTheDay)
+            self.delegate?.didSelectEntry(wordOfTheDay)
```

Comment:
@Br3nda it's magic - it tells Swift not to throw a runtime error if `self.delegate` is `nil`. The delegate is nil in the iPad view.